### ISSUE TITLE
Layout broken due to missing stylesheet

### ISSUE THREAD

**Issue by Aloso** [rust-lang/this-week-in-rust#2732]
A stylesheet isn't loaded on any page I tried. The HTML source contains these:

```html
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
<link rel="stylesheet" href="/">
```

The first one works, the second one is broken.

It seems that the `ASSET_URL` in [this line](https://github.com/rust-lang/this-week-in-rust/blob/master/themes/rusted/templates/base.html#L37) isn't defined. I looked in the git history, but there weren't any changes that could have caused this problem as far as I can tell.

P.S. sorry for reporting this 1 day before Christmas. Don't feel obliged to fix this in this year, and enjoy the holidays :)

**Comment by bjorn3** [comment]
In case anyone wants to patch it temporarily using the dev tools: The correct path to the stylesheet seems to be `/theme/css/web-min.css`.

**Comment by nellshamrell** [comment]
well, that's interesting. Taking a look and seeing if I can do a quick fix.

**Comment by nellshamrell** [comment]
Looks like this happened when I updated my Python environment (I use pelican to generate the website code), pushed a change which removed some duplicate links, and then synced those changes to the GitHub pages repo which hosts the website https://github.com/this-week-in-rust/this-week-in-rust.github.io/commit/e9501ceefed93e61f6a310ff18efed634b7b98bd

Interesting...

**Comment by nellshamrell** [comment]
Oddly enough, I'm seeing this issue even on fresh VM with a fresh install of all the dependencies. It's super strange.

**Comment by nellshamrell** [comment]
If anyone else would like to take a look at this (NO RUSH! Enjoy your holiday celebrations!), here is the process I use for generating the newsletter code (using pelican)

**Pre-reqs**
* Python 3
* Pip 3
* Sass gem `gem install sass`

**Instructions**
```
git clone https://github.com/rust-lang/this-week-in-rust.git
cd this-week-in-rust
pip3 install -r requirements.txt
pelican --delete-output-directory content
cd output; python3 -m http.server
```

Then open a browser and navigate to http://127.0.0.1:8000/blog/2021/12/22/this-week-in-rust-422/

It is generating `<link rel="stylesheet" href="/">` for the stylesheet, rather than something along the lines of `<link rel="stylesheet" href="/theme/css/web-min.css?84010e5b">`.

**Comment by Aloso** [comment]
I tried this. However, installing Markdown==2.4 failed (using Python 3.9). It complains that it uses the `imp` module which is deprecated. The solution is to use the Markdown==3.3.9 and removing the last line from `pelicanconf.py`. Then I was at least able to generate the pages without an error message. The stylesheet link is still broken, and I have no idea how to debug this. Google returns nothing relevant since the used dependencies are hopelessly out of date.

I managed to solve it by updating **all** dependencies to their latest version (Markdown `2.4 -> 3.3.9`, pelican `3.5 -> 4.7.1`, feedgenerator `1.7 -> 2.0`, webassets `0.11.1` -> pelical-webassets `2.0.0`), switching from Ruby sass to libsass and fixing the remaining errors.

I'll make a PR in a minute. Disclaimer: I do _not_ feel confident that this doesn't break any functionality.
